vulnz/src/docker/scripts/mirror.sh

```diff
@@ -48,6 +48,17 @@ function remove_lockfile() {
 }
 trap remove_lockfile SIGHUP SIGINT SIGQUIT SIGABRT SIGALRM SIGTERM SIGTSTP
 
-java $JAVA_OPT -jar /usr/local/bin/vulnz cve $DELAY_ARG $DEBUG_ARG $MAX_RETRY_ARG $MAX_RECORDS_PER_PAGE_ARG $CONTINUE_ARG --cache --directory /usr/local/apache2/htdocs
+attempt=1
+max_attempts=5
+
+while [ $attempt -le $max_attempts ]; do
+  java $JAVA_OPT -jar /usr/local/bin/vulnz cve $DELAY_ARG $DEBUG_ARG $MAX_RETRY_ARG $MAX_RECORDS_PER_PAGE_ARG $CONTINUE_ARG --cache --directory /usr/local/apache2/htdocs
+  exit_code=$?
+  if [ $exit_code -ne 100 ] && [ $exit_code -ne 101 ]; then
```

Comment:
we should hint what 100/101 means, what do you think. Looks magic here otherwise.

Reply:
also, you will need to use set +e around the while loop, or it will exit right away

Reply:
My bash isn't what it used to be - adding the `set +e` around the loop won't interfere with the `trap` in any adverse way?

Reply:
set -e 

above will lead to any command that has exit code > 0 to fatally exit right away

Using this as a default for any bash script is very good practice. But in your case, you want to handle such an error specifically, thus before running the while loop, you disable the 'auto exit' via set +e
so you can handle exit code > 0 yourself.

Reply:
(what you did in your patch was exactly right - thanks)